include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -6599,6 +6599,19 @@ FEEvaluation<dim,
   (void)dof_no;
   (void)first_selected_component;
 
+  Assert(
+    this->data->dofs_per_component_on_cell > 0,
+    ExcMessage(
+      "There is nothing useful you can do with an FEEvaluation "
+      "object with FE_Nothing, i.e., without DoFs! If you have passed "
+      "to MatrixFree::reinit() a collection of finite elements also containing "
```

Comment:
Can you break this line before 'containing' or, alternatively, use similar widths in all lines of the message?